engine/runtime/src/main/scala/org/enso/compiler/Compiler.scala

```diff
@@ -199,7 +199,7 @@ class Compiler(
           case None =>
             context.log(
               Level.SEVERE,
-              "Could not find entry point for compilation in package [{0}.{1}]",
+              "Could not find entry point for compilation in package [{}.{1}]",
```

Comment:
Either `{}` works or `{1}` works... I assume it is unlikely both formats would work at the same time.